nixos/release.nix

```diff
@@ -126,6 +126,30 @@ in rec {
       '';
     };
 
+  netboot.aarch64-linux = let build = (import lib/eval-config.nix {
```

Comment:
Don't you need to do something special here in case the `supportedSystems` parameter passed to the file doesn't include `aarch64`?

Reply:
Fixed